packages/bridge-ui/src/domain/token.ts

```diff
@@ -51,23 +51,32 @@ export const TKO: Token = {
   logoComponent: Tko,
 };
 
-export const HORSE: Token = {
-  name: "Horse Token",
+export const TEST_ERC20: Token = {
+  name: import.meta.env ? import.meta.env.VITE_TEST_ERC20_NAME_MAINNET : "Bull Token",
   addresses: [
     {
       chainId: CHAIN_MAINNET.id,
       address: import.meta.env
         ? import.meta.env.VITE_TEST_ERC20_ADDRESS_MAINNET
-        : "",
+        : "0x1B5Ccd66cc2408A0084047720167F6234Dc5498A",
     },
     {
       chainId: CHAIN_TKO.id,
       address: "0x00",
     },
   ],
   decimals: 18,
-  symbol: "HORSE",
+  symbol: import.meta.env ? import.meta.env.VITE_TEST_ERC20_SYMBOL_MAINNET : "BULL",
   logoComponent: Horse,
 };
 
-export const tokens = [ETH, HORSE];
+export interface TokenStore {
```

Comment:
can we not use the term `Store` here, since `svelte` itself uses the term `Store` to refer to a reactive state variable prefixed with a $ sign? can we use the term `TokenService` ?